lightning/src/chain/keysinterface.rs

```diff
@@ -94,19 +93,19 @@ impl_writeable_tlv_based!(DelayedPaymentOutputDescriptor, {
 	(12, channel_value_satoshis, required),
 });
 
-/// Information about a spendable output to our "payment key". See
-/// SpendableOutputDescriptor::StaticPaymentOutput for more details on how to spend this.
+/// Information about a spendable output to our "payment key".
+///
+/// See [`SpendableOutputDescriptor::StaticPaymentOutput`] for more details on how to spend this.
 #[derive(Clone, Debug, PartialEq, Eq)]
 pub struct StaticPaymentOutputDescriptor {
-	/// The outpoint which is spendable
+	/// The outpoint which is spendable.
 	pub outpoint: OutPoint,
-	/// The output which is referenced by the given outpoint
+	/// The output which is referenced by the given outpoint.
 	pub output: TxOut,
-	/// Arbitrary identification information returned by a call to
-	/// `Sign::channel_keys_id()`. This may be useful in re-deriving keys used in
-	/// the channel to spend the output.
+	/// Arbitrary identification information returned by a call to [`BaseSign::channel_keys_id`].
+	/// This may be useful in re-deriving keys used in the channel to spend the output.
 	pub channel_keys_id: [u8; 32],
-	/// The value of the channel which this transactions spends.
+	/// The value of the channel which this transaction spends.
```

Comment:
"The value of the funding channel output spent by this commitment transaction. This may be useful in re-deriving keys used in the channel to spend the output". This value isn't consumed directly by the signature digest as the output spent is a `to_remote` on a counterparty commitment transaction.

Reply:
Now went with "The value of the funding channel output which this transaction spends." to keep it concise. Let me know if that's alright.